x-pack/solutions/observability/plugins/synthetics/server/routes/monitor_cruds/add_monitor/add_monitor_api.ts

```diff
@@ -228,8 +229,9 @@ export class AddEditMonitorAPI {
     try {
       // we do this async, so we don't block the user, error handling will be done on the UI via separate api
       const defaultAlertService = new DefaultAlertService(context, server, savedObjectsClient);
-      defaultAlertService
-        .setupDefaultAlerts()
+      pRetry(async () => {
+        await defaultAlertService.setupDefaultAlerts();
+      })
```

Comment:
let's not do retry here, i will explain offline

Reply:
Reverted the commits containing the retries, they will be included in a different PR.